Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contest/ContestDetailsPage.tsx

```diff
@@ -59,22 +62,30 @@ const ContestDetailsPage = () => {
         [ contestTitle, setPageTitle ],
     );
 
-    const { contestId, participationType } = params;
+    const { contestId } = params;
 
     const contestIdToNumber = useMemo(
         () => Number(contestId),
         [ contestId ],
     );
 
     const isOfficial = useMemo(
-        () => participationType === ContestParticipationType.Compete,
-        [ participationType ],
+        () => {
+            if (isNil(contestDetails)) {
+                return null;
+            }
+
+            return contestDetails?.canBeCompeted;
+        },
+        [ contestDetails ],
     );
 
     const participantsCountByContestType = useMemo(
-        () => isOfficial
-            ? `Compete participants: ${contestDetails?.participantsCountByContestType}`
-            : `Practice participants: ${contestDetails?.participantsCountByContestType}`,
+        () => isNil(isOfficial)
+            ? ''
+            : isOfficial
+                ? `Compete participants: ${contestDetails?.participantsCountByContestType}`
```

Comment:
Isnt this going to put the same value both for practice and compete participants?

Reply:
This is changed in https://github.com/SoftUni-Internal/OpenJudgeSystem/pull/881.